diagrams-svg.cabal

```diff
@@ -61,6 +62,7 @@ Library
                      , lens                 >= 4.0   && < 4.12
                      , hashable             >= 1.1   && < 1.3
                      , optparse-applicative >= 0.10  && < 0.12
+                     , semigroups
```

Comment:
Although, it does seem like the upper bound is more likely to cause future breakage than to prevent it.